src/language/lang_id.php

```diff
@@ -0,0 +1,659 @@
+    'Semua Tim',
+  'Protected' =>
+    'Protected',
+  'Score' =>
+    'Nilai',
+  'Change Password' =>
+    'Ubah Password',
+  'Admin Level' =>
+    'Admin Level',
+  'Visibility' =>
+    'Visibility',
+  'Team Management' =>
+    'Pengaturan Tim',
+  'Add Team' =>
+    'Tambah Tim',
+  'None' =>
+    'Tidak Ada',
+  'Logo Name' =>
+    'Nama Logo',
+  'Logo Management' =>
+    'Pengaturan Logo',
+  'Session' =>
+    'Session',
+  'Cookie' =>
+    'Cookie',
+  'Creation Time' =>
+    'Waktu Pembuatan',
+  'Last Access' =>
+    'Diakses Terakhir',
+  'Data' =>
+    'Data',
+  'Sessions' =>
+    'Session',
+  'entry' =>
+    'entry',
+  'No Entries' =>
+    'Tidak Ada Catatan',
+  'Game Logs' =>
+    'Catatan Permainan',
+  'Game Logs Timeline' =>
+    'Jangka Waktu Catatan',
+  'End Game' =>
+    'Akhiri Permainan',
+  'Begin Game' =>
+    'Mulai Permainan',
+  'Game Admin' =>
+    'Admin Permainan',
+  'Controls' =>
+    'Kontrol',
+  'Quiz' =>
+    'Kuis',
+  'Flags' =>
+    'Flag',
+  'Bases' =>
+    'Base',
+  'Countries' =>
+    'Negara',
+  'Logos' =>
+    'Logo',
+  //Translations for inc/* and inc/gameboard/*
+  'captured' =>
+    'menguasai',
+  'Status' =>
+    'Status',
+  'Completed' =>
+    'Selesai',
+  'Remaining' =>
+    'Tersisa',
+  'Start' =>
+    'Mulai',
+  'End' =>
+    'Akhiri',
+  'Rank' =>
+    'Ranking',
+  'pts' =>
+    'point', //points
+  'Your Rank' =>
+    'Ranking Kamu',
+  'Your Score' =>
+    'Nilai Kamu',
+  'Everyone' =>
+    'Semua Orang',
+  'Your Team' =>
+    'Tim Kamu',
+  'Captured' =>
+    'Menguasai',
+  'Initiating' =>
+    'Memulai',
+  'run : > boot_sequence' =>
+    'run : > boot_sequence',
+  'Extracting' =>
+    'Mengekstrak',
+  //Translations for Utils.php's time_ago() function
+  'just now' =>
+    'baru saja',
+  'd' =>
+    'hari', //day
+  'hr' =>
+    'jam', //hour
+  'min' =>
+    'menit', //minute
+  'sec' =>
+    'detik', //second
+  'ds' =>
+    'hari', //days
+  'hrs' =>
+    'jam', //hours
+  'mins' =>
+    'menit', //minutes
+  'secs' =>
+    'detik', //seconds
+  'ago' =>
+    'yang lalu',
+  //Translations for ModalControllers
+  'begin_' =>
+    'mulai_',
+  'Are you sure you want to kick off the game? Logs will be cleared and progressive scoreboard will start' =>
+    'Apakah kamu yakin ingin memulai permainan? Catatan permainan akan dihapus dan papan nilai akan dimulai',
+  'end_' =>
+    'akhir_',
+  'Are you sure you want to finish the current game?' =>
+    'Apakah kamu yakin ingin mengakhiri permainan yang sedang berlangsung?',
+  'Are you sure you want to logout from the game?' =>
+    'Apakah kamu yakin ingin keluar dari permainan?',
+  'Saved' =>
+    'Tersimpan',
+  'All changes have been successfully saved.' =>
+    'Semua perubahan telah berhasil disimpan.',
+  'Error' =>
+    'Error',
+  'Sorry your form was not saved. Please correct the all errors and save again.' =>
+    'Gagal menyimpan. Mohon perbaiki semua erro yang terjadi dan cobalah menyimpan kembali.',
```

Comment:
Suggested Change: 'Formulir kamu tidak disimpan.'

Comement: Context mistranslation of the sentence. It seemed that the form was not saved due to technical reasons."Gagal menyimpan" = "Fail to save".